cypress/integration/calls.ts

```diff
@@ -308,7 +308,7 @@ context('Calls tests', () => {
       .click();
 
     cy.get('[title="Delete"]')
-      .first()
+      .eq(1)
```

Comment:
Maybe this can be targeted better and instead of getting second delete button in the view we can have something like: 
`cy.get('some_better_selector [title="Delete"]').first().click()`

Reply:
I agree that is much better